deployment/tasks/after_update_code.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This has to be mended to match current (docker & make-base) project build, compare https://github.com/wmde/FundraisingFrontend/blob/master/Makefile